### PR TITLE
Adding CoC to repo

### PR DESCRIPTION
**Pull request by saisandhya3198**
#1 
Please review the attached CoC file and come up with add-ons and updates! 
Thanks.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+			Gayatri Vidya Parishad -Artificial Intelligence CLUB
```
Comment (sakethramanujam) [inline CODE_OF_CONDUCT.md]: Use markdown headings!

Comment (sakethramanujam) [inline CODE_OF_CONDUCT.md]: Use Markdown syntax

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Community Code of Conduct
```
Comment (sakethramanujam) [inline CODE_OF_CONDUCT.md]: `##`

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+The community and its members must, at all times, comply with all applicable rules and regulations. The community will not condone the activities of members who achieve results through violation of the rule or unethical practices.
+
+ Expected Behaviour:
+The community expects its members to conduct themselves in a professional manner. 
+
+The community expects that members will perform their duties conscientiously, honestly, and in accordance with the best interests of the community. 
+
+All members share a serious responsibility for the community’s good public relations, especially at the community level. Their readiness to help with religious, charitable, educational, and civic activities brings credit to the community and is encouraged. 
+
+Members who have access to community funds in any form must follow the prescribed procedures for recording, handling, and protecting money as detailed in the community’s policies and procedures or other explanatory materials, or both.
```
Comment (sakethramanujam) [inline CODE_OF_CONDUCT.md]: Good thing brought up! We don't have a policy for this yet.

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+ Expected Behaviour:
+The community expects its members to conduct themselves in a professional manner. 
+
+The community expects that members will perform their duties conscientiously, honestly, and in accordance with the best interests of the community. 
+
+All members share a serious responsibility for the community’s good public relations, especially at the community level. Their readiness to help with religious, charitable, educational, and civic activities brings credit to the community and is encouraged. 
+
+Members who have access to community funds in any form must follow the prescribed procedures for recording, handling, and protecting money as detailed in the community’s policies and procedures or other explanatory materials, or both. 
+
+When handling financial and personal information about members or others with whom the community has dealings, the following principles are to be abided:
```
Comment (sakethramanujam) [inline CODE_OF_CONDUCT.md]: sure that the word `abided` is right?

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Members who have access to community funds in any form must follow the prescribed procedures for recording, handling, and protecting money as detailed in the community’s policies and procedures or other explanatory materials, or both. 
+
+When handling financial and personal information about members or others with whom the community has dealings, the following principles are to be abided: 
+- Collect, use, and retain only the personal information necessary for the community’s business. Whenever possible, obtain any relevant information directly from the person concerned. Use only reputable and reliable sources to supplement this information. 
+- Retain information only for as long as necessary or as required by law. Protect the physical security of this information. 
+- Limit internal access to personal information to those with a legitimate business reason for seeking that information. Only use personal information for the purposes for which it was originally obtained. Obtain the consent of the person concerned before externally disclosing any personal information, unless legal process or contractual obligation provides otherwise. 
+
+
+Unacceptable Behaviour:
```
Comment (sakethramanujam) [inline CODE_OF_CONDUCT.md]: Markdown heading again

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+Community funds and all other assets of the community are proposed for the community only and not for personal benefit. This includes the personal use of community based assets, such as computers. It is strictly prohibited.
+
+Members must take care to separate their personal roles from their community positions when communicating on matters not involving community business. members must not use community identification, stationery, supplies, and equipment for personal or political matters.
+
+When communicating publicly on matters that involve community business, members must not presume to speak for the community on any topic, unless they are certain that the views they express are those of the community, and it is the community’s desire that such views be publicly disseminated.
+
+When dealing with anyone outside the community, including public officials, members must take care not to compromise the integrity or damage the reputation of either the community, or any outside individual, business, or government body.
+
+How to report:
```
Comment (sakethramanujam) [inline CODE_OF_CONDUCT.md]: Where is the core committee situated, how does a freshly onboarded individual know who's who!

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+Violation of any of the above rules should immediately be brought to the notice of the core committee.It can be done through a letter,an email or a personal appointment with any of the committee members. Such approaches will be much appreciated and will help to protect the values on which  this community was built in the first place.
+
+Consequences:
```
Comment (sakethramanujam) [inline CODE_OF_CONDUCT.md]: Nice!

**Review by sakethramanujam — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (saisandhya3198) [reply CODE_OF_CONDUCT.md]: done!

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (saisandhya3198) [reply CODE_OF_CONDUCT.md]: Yeah, should work on that

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (saisandhya3198) [reply CODE_OF_CONDUCT.md]: checked it with Google. :wink:

File: CODE_OF_CONDUCT.md (hunk 6 above)
Comment (saisandhya3198) [reply CODE_OF_CONDUCT.md]: The current organisers of the club are tentatively the core committee. And they are all approachable to the members.

**Review by sakethramanujam — APPROVED** [review]
LGTM! Merging as v 0.1

**Pull request merged**